src/main/java/teammates/ui/template/AdminActivityLogTableRow.java

```diff
@@ -0,0 +1,211 @@
+package teammates.ui.template;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.TimeZone;
+
+import teammates.common.util.ActivityLogEntry;
+import teammates.common.util.Const;
+import teammates.common.util.TimeHelper;
+import teammates.common.util.Url;
+
+public class AdminActivityLogTableRow {
+    
+    private static final int TIME_TAKEN_WARNING_LOWER_RANGE = 10000;
+    private static final int TIME_TAKEN_WARNING_UPPER_RANGE = 20000;
+    private static final int TIME_TAKEN_DANGER_UPPER_RANGE = 60000;
```

Comment:
This was the way before, indeed, but "warning" and "danger" are concepts borrowed by Bootstrap classes. Let's not mix them up in Java.
If I understand correctly, `warning` is used when the time taken is quite long (10-20s) and `danger` when it is long (20-60s). How the numbers are determined are beyond my knowledge, but let's not talk about that. The more important thing is, the variable names should be independent of its subsequent usage.
Consider `TIME_ELAPSED_FAST`, `TIME_ELAPSED_MODERATE`, `TIME_ELAPSED_TIMEOUT`; you might be able to think of better names, but the idea is you can immediately tell what the variables meant by looking at the name, and they are independent of the "warning" and "danger" concepts.